contracts/event/MultipleResultsEvent.sol

```diff
@@ -137,6 +141,10 @@ contract MultipleResultsEvent is NRC223Receiver, Ownable {
         require(
             resultSetEndTime > resultSetStartTime,
             "resultSetEndTime should be > resultSetStartTime");
+        require(arbitrationLength > 0, "arbitrationLength should be > 0");
```

Comment:
I think arbitration length and arbitration threshold must be enumerated.
Create a arbitration_lengths = [48, 24, 12, 6], arbitration_thresholds = [100, 1000, 5000, 10000].
Then pass an index as argument so that arbitration_length = arbitration_lengths[index], arbitration_thresdhold = arbitration_thresholds[index].

Reply:
This is handled, L170 calculateThreshold